framework/src/mesh/MooseMesh.C

```diff
@@ -191,6 +199,21 @@ MooseMesh::MooseMesh(const InputParameters & parameters) :
       _partitioner_name = "parmetis";
       _partitioner_overridden = true;
     }
+
+    // Add geometric ghosting functors to mesh if running with DistributedMesh
+    if (getParam<bool>("ghost_point_neighbors"))
+      _ghosting_functors.emplace_back(libmesh_make_unique<GhostPointNeighbors>(*_mesh));
+
+    auto num_edge_neighbors = getParam<unsigned short>("num_ghosted_edge_neighbors");
+    if (num_edge_neighbors > 1)
+    {
+      auto default_coupling = libmesh_make_unique<DefaultCoupling>();
+      default_coupling->set_n_levels(num_edge_neighbors);
+      _ghosting_functors.emplace_back(std::move(default_coupling));
+    }
```

Comment:
@roystgnr - This logic allows me to add both "point" and "edge" functors to the `MeshBase` object. However, when I tested them together, it appears that the second one didn't run (i.e. I received the output below with the thinner banding between partitions). I might be making a mistake somewhere, but I wanted to make sure I was using these correctly before digging in.